src/client/network/nusight_network.ts

```diff
@@ -33,16 +87,16 @@ export class NUsightNetwork {
   public onNUClearMessage<T>(messageType: MessageType<T>, cb: MessageCallback<T>) {
     const messageTypeName = this.messageTypePath.getPath(messageType)
     return this.nuclearnetClient.on(`NUsight<${messageTypeName}>`, (packet: NUClearNetPacket) => {
-      // Remove NUsight header for decoding by protobufjs
-      const buffer = new Uint8Array(packet.payload).slice(HEADER_SIZE)
-      const message = messageType.decode(buffer)
```

Comment:
I'm just going to send the message type as a string and let it work it out itself using the message_type_names.ts